sites/hashai/resources/docs/simulation/tutorials/hello-hash.md

```diff
@@ -38,17 +38,17 @@ Beneath the workspace's right-hand view-pane, you'll find the simulation control
 
 However, if you toggle back from the raw output view to the 3D viewer, you may notice... nothing at all. Alice and Bob haven't been rendered, because they haven't been given a position in space. Let's go ahead and fix that.
 
-```javascript
+```json
 [
   {
-    agent_name: "Alice",
-    position: [0, 0],
+    "agent_name": "Alice",
+    "position": [0, 0]
   },
   {
-    agent_name: "Bob",
-    position: [2, 0],
-  },
-];
+    "agent_name": "Bob",
+    "position": [2, 0]
+  }
+]
```

Comment:
The section below also refers to the `init` file and is written in `JS`. 

Maybe we should add three tabs (to include a JSON one) or maybe we should actually just get rid of these two types because we haven't explained about `init.js` or `init.py` here yet.

Reply:
Can we add the 3x tabs? I think preferable here. Ty for the flag.

Reply:
Let's make the JSON one the default then just to avoid further confusion if we can.

Reply:
I did a deeper look and it's most probably not worth us keeping the tabs. They only slightly differ in the syntax, but the code is not correct, it's just the array. As init behaviors will be introduced later, removing the tabs probably makes the most sense. The page just introduces normal behaviors. Introducing init behaviors at the same time may be a little bit too much.

https://hash.ai/docs/simulation/tutorials/hello-hash